### PR TITLE
Add contributions guidelines for Tartiflette

### PR DESCRIPTION
**Pull request by tsunammis**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+* [Code of Conduct](#code-of-conduct)
+* [How can I contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Pull Requests](#pull-requests)
+* [Changelog](#changelog)
+
+## Code of conduct
+
+We expect from each contributor a respect for the [Code of Conduct](./CODE-OF-CONDUCT.md) which governs the **Tartiflette** project.
```
Comment (achedeuzot) [inline docs/CONTRIBUTING.md]: We expect from each contributor to respect the [Code of Conduct]

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## How can I contribute?
+### Reporting bugs
+
+To report a bug, you have to [create an issue on github](https://github.com/dailymotion/tartiflette/issues/new) and follow the following guidelines.
```
Comment (achedeuzot) [inline docs/CONTRIBUTING.md]: To report a bug, please [create an issue on github](https://github.com/dailymotion/tartiflette/issues/new) and follow the guidelines below.

File: docs/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+* [ ] **Explain with a simple sentence the behavior expected**
+* [ ] **Is a regression from a previous versions?** _e.g Yes_
+
+### Suggesting enhancements
+
+* A feature is missing from the GraphQL Specification?
+* A hook is missing in the API to plug your code?
+* A part of the project could be improved?
+
+Don't hesitate to ask a new feature or code it by yourself.
```
Comment (achedeuzot) [inline docs/CONTRIBUTING.md]: Don't hesitate to ask a new feature or join in and send us a pull request with your own feature !

File: docs/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+* **Stage 2 - Draft:** Push your code & create a pull-request with the `WHY` of your work.
+* **Stage 3 - Candidate:** Stable code, tested & documented.
+* **Stage 4 - :tada:** Your code is merged and available.
+
+### Pull Requests
+
+Before submitting your pull-request, make sure the following is done.
+
+* [ ] Fork [the repository](https://github.com/dailymotion/tartiflette) and create your branch from `master` so that it can be merged easily.
+* [ ] Update CHANGELOG.md with your change (include reference to issue & this PR).
```
Comment (achedeuzot) [inline docs/CONTRIBUTING.md]: * [ ] Update CHANGELOG.md with your change (include reference to issue & your pull request).

File: docs/ISSUE_TEMPLATE.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+Do you want to **report a bug?** or **request a feature?** 
+
```
Comment (achedeuzot) [inline docs/ISSUE_TEMPLATE.md]: *Please keep the corresponding template*

File: docs/ISSUE_TEMPLATE.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+## Report a bug
+
+Please provide the steps to reproduce and if possible a reproductible environment. **As we are working directly with container, please provide the Dockerfile sample or the Docker image name**
```
Comment (achedeuzot) [inline docs/ISSUE_TEMPLATE.md]: Please provide the steps to reproduce your problem and, if possible, a full reproducible environment. **As we are working directly with containers, please provide the Dockerfile sample or the Docker image name**

File: docs/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+* [ ] **Explain with a simple sentence the behavior expected**
```
Comment (achedeuzot) [inline docs/ISSUE_TEMPLATE.md]: the expected behavior OR the behavior you expected

File: docs/ISSUE_TEMPLATE.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+* [ ] **Is a regression from a previous versions?** _e.g Yes_
+* [ ] **Stacktrace** _e.g gist, pastebin..._
+
+## Request a feature
+
+* A feature is missing from the GraphQL Specification?
+* A hook is missing in the API to plug your code?
+* A part of the project could be improved?
+
+Don't hesitate to ask a new feature or code it by yourself.
```
Comment (achedeuzot) [inline docs/ISSUE_TEMPLATE.md]: Don't hesitate to ask a new feature or join in and send us a pull request with your own feature !

**Comment by tsunammis** [comment]
@achedeuzot it should be good.

**Pull request merged**